core/src/main/java/io/cucumber/core/runner/TimeService.java

```diff
@@ -1,18 +1,16 @@
 package io.cucumber.core.runner;
 
+import java.time.Instant;
+
 public interface TimeService {
```

Comment:
Can you replace this with `Clock`.

Reply:
You mean use `Clock` instead of `Instant` everywhere?

Because that would mean the `TimeService` would return a `clock()` instance instead of `instant()` and same with the `EventBus` and all others (`getClock()` instead of `getInstant()`).

Reply:
No. I mean replace `TimeService` with `Clock`. They serve the same purpose.

Reply:
I removed `TimeService` and updated all the references to use `Clock.fixed()`or `Clock.systemUTC()` instead. Theres a new `ClockStub` now too. Can you take a look if its fine now.